tests/conftest.py

```diff
@@ -1,117 +1,57 @@
 import pytest
 from brownie import config
-
-@pytest.fixture
-def andre(accounts):
-    # Andre, giver of tokens, and maker of yield
-    yield accounts[0]
-
-
-@pytest.fixture
-def token(andre, Token):
-    yield andre.deploy(Token)
+from brownie import Contract
 
 
 @pytest.fixture
 def gov(accounts):
-    # yearn multis... I mean YFI governance. I swear!
-    yield accounts[1]
+    yield accounts[0]
 
 
 @pytest.fixture
-def rewards(gov):
-    yield gov  # TODO: Add rewards contract
+def rewards(accounts):
+    yield accounts[1]
 
 
 @pytest.fixture
 def guardian(accounts):
-    # YFI Whale, probably
     yield accounts[2]
 
 
 @pytest.fixture
-def vault(pm, gov, rewards, guardian, token):
-    Vault = pm(config["dependencies"][0]).Vault
-    vault = guardian.deploy(Vault, token, gov, rewards, "", "")
-    yield vault
-
-
-@pytest.fixture
-def strategist(accounts):
-    # You! Our new Strategist!
+def management(accounts):
     yield accounts[3]
 
 
 @pytest.fixture
-def keeper(accounts):
-    # This is our trusty bot!
+def strategist(accounts):
     yield accounts[4]
 
 
 @pytest.fixture
-def strategy(strategist, keeper, vault, Strategy):
-    strategy = strategist.deploy(Strategy, vault)
-    strategy.setKeeper(keeper)
-    yield strategy
-
-
-@pytest.fixture
-def nocoiner(accounts):
-    # Has no tokens (DeFi is a ponzi scheme!)
+def keeper(accounts):
     yield accounts[5]
 
 
 @pytest.fixture
-def pleb(accounts, andre, token, vault):
-    # Small fish in a big pond
-    a = accounts[6]
-    # Has 0.01% of tokens (heard about this new DeFi thing!)
-    bal = token.totalSupply() // 10000
-    token.transfer(a, bal, {"from": andre})
-    # Unlimited Approvals
-    token.approve(vault, 2 ** 256 - 1, {"from": a})
-    # Deposit half their stack
-    vault.deposit(bal // 2, {"from": a})
-    yield a
+def token():
+    token_address = "0x6b175474e89094c44da98b954eedeac495271d0f"  # this should be the address of the ERC-20 used by the strategy/vault
+    yield Contract.from_explorer(token_address)
```

Comment:
`Contract(token_address)` since we have `autofetch_sources: True`

Reply:
This change will make sure that contracts pulled from Etherscan are cached
```suggestion
    yield Contract(token_address)
```